vNext/src/Orleans/Shims/ClassesForTesting/CustomSerializerTestClasses.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using Orleans.CodeGeneration;
+using Orleans.Serialization;
+
+namespace UnitTests.CustomSerializerTestClasses
```

Comment:
This is for CustomSerializerTests class. I added those types to AppDomain.GetAssemblely() method so SerializationManager can map serializer for them. They are in shim folder because they are related to AppDomain. Create them in other project would end up add reference for Orleans project.